### PR TITLE
Add headset pose publishing

### PR DESCRIPTION
**Pull request by joshanderson-kw**
Depends on #214

### CODE REVIEW

File: ros/angel_system_nodes/angel_system_nodes/hl2ss_ros_bridge.py [hunk 1]
```diff
@@ -270,6 +270,16 @@ def connect_hl2ss_pv(self) -> None:
...
 
+        # Get the camera parameters for this configuration
+        pv_cam_params = hl2ss.download_calibration_pv(
+            self.ip_addr,
+            self.pv_port,
+            self.pv_width,
+            self.pv_height,
+            self.pv_framerate
+        )
+        self.camera_intrinsics = [float(x) for x in pv_cam_params.intrinsics.flatten()]
```
Comment (Purg) [inline ros/angel_system_nodes/angel_system_nodes/hl2ss_ros_bridge.py:281]: Do we know if this is "correct"? I'm recalling we had that trouble when we switched from a deprecated MS plugin where the intrinsic of the camera became incorrect and we had to hard-code some matrix.

Comment (joshanderson-kw) [reply ros/angel_system_nodes/angel_system_nodes/hl2ss_ros_bridge.py:287]: No, unfortunately it doesn't match our hard-coded values. It's the values for focal length are -985 and 984, whereas our values are `FOCAL_LENGTH_X = 1.630`, `FOCAL_LENGTH_Y = 2.5084`.

I wonder if there is some unit conversion I am just not understanding.

Comment (Purg) [reply ros/angel_system_nodes/angel_system_nodes/hl2ss_ros_bridge.py:287]: I suppose let's keep the hard-coded intrinsics in our back pocket if we find that we are not able to use this intrinsics value correctly.

**Review by Purg — APPROVED** [review]
LGTM pending dependency merger.

**Comment by joshanderson-kw** [comment]
> LGTM pending dependency merger

Conflicts should be fixed here now.

**Pull request merged**